ModelCatalogueGenomicsPlugin/grails-app/services/modelcataloguegenomicsplugin/GelJsonService.groovy

```diff
@@ -0,0 +1,160 @@
+package modelcataloguegenomicsplugin
+
+
+import grails.transaction.Transactional
+import org.modelcatalogue.core.CatalogueElement
+import org.modelcatalogue.core.DataClass
+
+
+//@Transactional
+class GelJsonService {
```

Comment:
I would prefer using [Groovy JsonBuilder](http://docs.groovy-lang.org/latest/html/gapi/groovy/json/JsonBuilder.html) to String concatenation as it also escapes the values properly.